scprep/run/dyngen.py

```diff
@@ -0,0 +1,241 @@
+import pandas as pd
+from . import r_function
+
+_install = r_function.RFunction(
+    args="""lib=.libPaths()[1], dependencies=NA, verbose=TRUE""",
+    body="""
+        cat ('Installing package in:', lib, '\n')
+        quiet <- !verbose
+
+        if (!require('remotes')) install.packages('remotes')
+        remotes::install_github('dynverse/dyngen',
+                         repos = "http://cran.us.r-project.org",
+                         lib=lib, dependencies=dependencies,
+                         quiet=quiet)
+    """,
+)
+
+_get_backbones = r_function.RFunction(
+    args="""lib=.libPaths()[1]""",
+    body="""
+        library(dyngen, lib.loc=lib)
+        names(list_backbones())
+    """,
+)
+
+_DyngenSimulate = r_function.RFunction(
+    args="""
+        backbone_name=character(), num_cells=500, num_tfs=100, num_targets=50, num_hks=25,
+        simulation_census_interval=10, compute_cellwise_grn=FALSE, compute_rna_velocity=FALSE,
+        n_jobs=7, random_state=NA, lib=.libPaths()[1], verbose=TRUE
+    """,
+    body="""
+        library(dyngen, lib.loc=lib)
+        if (!(backbone_name %in% names(list_backbones()))) {
+            stop("Input not in list of dyngen backbones. Choose name from get_backbones().")
+        }
+        if (!is.na(random_state)) {
+            set.seed(random_state)
+        }
+        
+        backbones <- list('bifurcating'=backbone_bifurcating(),
```

Comment:
Is there a cleaner way of getting this list via `get_backbones()`? Can we somehow ensure that this hardcoded list doesn't end up out of date?